### ISSUE TITLE
RPCS3 No Longer Opens After Update

### ISSUE THREAD

**Issue by mccorkled** [RPCS3/rpcs3#16257]
### Quick summary

Been running RPCS3 off of a NAS for over 6 years with no issues. Updated today like I always do via the GUI and RPCS3 never reopened.  Updated GPU drivers, and restarted PC, with the same result. Opens off of the C drive fine, but trying to open via the NAS, RPCS3 goes into the "background processes" within Task Manager and never opens. 

I have no log due to it never opening. 

Synology NAS, Windows 11 and everything is updated. 

Several other users in the Discord are reporting the same issue. Appears to be happening when running RPCS3 off of any drive other than C:

**Comment by elad335** [comment]
I'll reopen when you post on which RPCS3 it started.

**Comment by mccorkled** [comment]
As XvPhoenix pointed out in the Discord, it works pre 17061

**Comment by GreemDev** [comment]
> Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕

Oh no, what will they ever do without your valuable efforts opening invalid issues.

**Comment by mccorkled** [comment]
> > Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕
> 
> Oh no, what will they ever do without your valuable efforts opening invalid issues.

Found the clown

**Comment by mccorkled** [comment]
> > Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕
> 
> Oh no, what will they ever do without your valuable efforts opening invalid issues.

Imagine being so triggered by someone finding a bug that you don't agree with (even after multiple people in the same night report it), that you not only have to act like an asshole in Discord, but also have to take to commenting on the Github issues. #sad

**Comment by GreemDev** [comment]
> > > Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕
> > 
> > 
> > Oh no, what will they ever do without your valuable efforts opening invalid issues.
> 
> Imagine being so triggered by someone finding a bug that you don't agree with (even after multiple people in the same night report it), that you not only have to act like an asshole in Discord, but also have to take to commenting on the Github issues. #sad

It's really fucking hilarious that you assume it's me.

**Comment by mccorkled** [comment]
> > > > Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕
> > > 
> > > 
> > > Oh no, what will they ever do without your valuable efforts opening invalid issues.
> > 
> > 
> > Imagine being so triggered by someone finding a bug that you don't agree with (even after multiple people in the same night report it), that you not only have to act like an asshole in Discord, but also have to take to commenting on the Github issues. #sad
> 
> It's really fucking hilarious that you assume it's me.

I agree, your behavior is quite hilarious.

**Comment by GreemDev** [comment]
> > > > > Banned from the Discord because someone wanted to troll me on an issue that she didn't believe existed, and I proved did. LOL Typical emulator BS. You've lost my monetary contributions, and free help 🖕
> > > > 
> > > > 
> > > > Oh no, what will they ever do without your valuable efforts opening invalid issues.
> > > 
> > > 
> > > Imagine being so triggered by someone finding a bug that you don't agree with (even after multiple people in the same night report it), that you not only have to act like an asshole in Discord, but also have to take to commenting on the Github issues. #sad
> > 
> > 
> > It's really fucking hilarious that you assume it's me.
> 
> I agree, your behavior is quite hilarious.

You are so oblivious it's not even funny.

**Comment by Spartan304** [comment]
For what it's worth, I think the issue is with a buggy Windows Defender SmartScreen.  When running an app from the shared drive, it's not popping up with the SmartScreen prompt, and even yesterday during testing, it wasn't prompting on the copies I had extracted to my C drive.  Just a bit ago though, I snagged 17066, tried launching it from the shared folder, no dice.  Extracted it to C drive, ran it, and... SmartScreen prompt.  I allowed it, closed it, went back to the copy on the shared drive and IT LAUNCHED.  Thanks Microsoft....
